Wordle.UI/Program.cs

```diff
@@ -8,13 +8,51 @@ public static class Program
 {
     public static void Main(string[] args)
     {
-        try {
-            Console.WriteLine("Hello");
-        }
+        try
+        {
+            var game = new Game();
+            Console.Clear();
+            Console.WriteLine(@"

```

Comment:
future refactoring idea - create a structure for the rows and row lines.

Reply:
That's the thought! I think my next commit will have private classes for DisplayRow and DisplayCell that can construct a board.